moderation.php

```diff
@@ -2315,9 +2315,12 @@
 			}
 		}
 
-		if(empty($posts))
-		{
-			error($lang->error_inline_nopostsselected, $lang->error);
+		if(count($posts) < 2) {
```

Comment:
Nitpick: code style:

```suggestion
		if(count($posts) < 2)
		{
```